character-sfx.js

```diff
@@ -342,6 +359,74 @@ class CharacterSfx {
   
     };
     _handleNarutoRun();
+
+    // combo
+    const dispatchComboSoundEvent = (combo, indexOfCombo) =>{
+      this.dispatchEvent(new MessageEvent('comboSoundEmit', {
+        data: {
+          combo: combo,
+          indexOfCombo: indexOfCombo
+        },
+      }));
+    }
+    const _handleCombo = () => {
+      let currentCombo;
+      if(this.player.hasAction('use') && this.player.getAction('use').behavior === 'sword'){
+        currentCombo = this.player.getAction('use').animation ? 
+          this.player.getAction('use').animation // sword
+          : 
+          this.player.getAction('use').animationCombo[this.player.avatar.useAnimationIndex] // silsword

```

Comment:
Why is this mentioning app names?

Reply:
Got it. Because there are 2 combo animations(silsword and sword). Will remove these comments